content/docs/command-reference/exp/list.md

```diff
@@ -45,3 +46,51 @@ options below).
 
 - `-v`, `--verbose` - displays detailed tracing information from executing the
   `dvc pull` command.
+
+## Examples
+
+> This example is based on our
+> [Get Started](/doc/tutorials/get-started/experiments), where you can find the
+> actual source code.
+
+Let's say we have run 3 experiments in our project. You can quickly list the
+available experiments with this command:
+
+```dvc
+$ dvc exp list --all
+10-bigrams-experiment:
+        exp-e6c97
+        exp-1dad0
+        exp-1df77
+```
+
+> Contrast this with the full table
+> [displayed by `dvc exp show`](/doc/command-reference/exp/show#examples).
+
+You can also list experiments in a any DVC repo with `dvc exp list`:
+
+```dvc
+$ dvc exp list --all git@github.com:iterative/example-get-started.git
```

Comment:
Do we need to actually push some experiments to the repo to support this? I don't it will work right now, which could cause confusion.

Reply:
Maybe to one of the exp-specific repos you published instead (for the next iteration). Do they already have some exps up?

Reply:
I haven't pushed any experiments since they need a data remote to save the experiment data. However, there is a read-only remote associated with `example-get-started`, so I could add some there. Let me know what you think.

Reply:
Yep, having some exps in there would be nice. That remote is read-only in it's HTTP form but you can use the S3 config to upload stuff as you probably already know from modifying the generator script 🙂

Reply:
Sounds good, I just pushed some experiments up to https://github.com/iterative/example-get-started.

Reply:
Amazing, thanks! 🙏